src/routes/create/+page.svelte

```diff
@@ -1,10 +1,10 @@
 <script lang="ts">
-import type { ActionData } from './$types';
+  import type { ActionData } from './$types';
   import { Search, Button } from 'flowbite-svelte';
 	export let form: ActionData;
+  import Icon from '@iconify/svelte';
```

Comment:
move this up to line 4 so it's with the other import statements. then move what's on line 4 down two lines